pydantic/_internal/_config.py

```diff
@@ -154,7 +154,7 @@ def __getattr__(self, name: str) -> Any:
                 except KeyError:
                     raise AttributeError(f'Config has no attribute {name!r}') from None
 
-    def core_config(self, obj: Any) -> core_schema.CoreConfig:
+    def core_config(self, title: str | None) -> core_schema.CoreConfig:
         """Create a pydantic-core config, `obj` is just used to populate `title` if not set in config.
```

Comment:
Could you please update these docs? Also, we'll want to reflect the fact that the `title` from `config` takes prio.